File: design-patterns/go/abstract_factory/transport/truck.go
// truck.go

package main

import "fmt"

type Truck struct {
}

func NewTruck() (truck *Truck) {
	truck = &Truck{}
	return
}

func (truck *Truck) Start() {
	fmt.Println("Truck Started")
}

func (truck *Truck) Stop() {
	fmt.Println("Truck Stopped")
}

func (truck *Truck) Repair() {
	fmt.Println("Truck Repair")
}

File: design-patterns/php/abstractfactory/transport/Truck.php
<?php
// Truck.php

namespace BigBoxCode\DesignPattern\AbstractFactory\Transport;

class Truck implements Transport {
    public function start(): void {
        echo "Truck Started\n";
    }

    public function stop(): void {
        echo "Truck Stopped\n";
    }

    public function repair(): void {
        echo "Truck Repair\n";
    }
}